pkg/descheduler/strategies/nodeutilization/nodeutilization.go

```diff
@@ -82,8 +82,15 @@ func getNodeUsage(
 	lowThreshold, highThreshold api.ResourceThresholds,
 	resourceNames []v1.ResourceName,
 	getPodsAssignedToNode podutil.GetPodsAssignedToNodeFunc,
+	useDeviationThresholds bool,
 ) []NodeUsage {
 	var nodeUsageList []NodeUsage
+	var averageCPUUsagePercent float64
```

Comment:
`averageResourceUsagePercent := map[v1.ResourceName]float64{}` will allow to iterate over resources (including the extended ones)